src/components/application_manager/test/commands/mobile/put_file_test.cc

```diff
@@ -344,6 +345,53 @@ TEST_F(PutFileRequestTest, Run_InvalidPutFile_UNSUCCESS) {
   command->Run();
 }
 
+TEST_F(PutFileRequestTest, Run_SrcSumEqual_SendSuccessResponse) {
```

Comment:
@BSolonenko 
- CRC
- It is not obvious why result is successful. How do we know that binary data is not empty?

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2139/commits/17cae31dfeb5a8e55281633ebe3efc6dd051c614